modules/ob-release-notes-1-5.adoc

```diff
@@ -47,16 +47,12 @@ If you specify a commit SHA as a revision, the full history is always cloned bef
 [id="fixed-issues-1-5_{context}"]
 == Fixed issues
 
-The following list highlights fixed issues in {builds-shortname} 1.5.
-
 * Before this update, the build controller had permission to modify `CustomResourceDefinitions`. This caused the controller to have excessive permissions, resulting in a minor security risk. With this update, the issue is fixed.
 Before this update, if the {builds-operator} failed to reconcile the `ShipwrightBuild` or `SharedResource` objects but the status update of the build controller succeeded, the controller was not requeued for reconciliation. With this update, the issue is resolved, and the controller is correctly requeued for reconciliation after a failure.
 
 [id="deprecated-features-1-5_{context}"]
 == Deprecated features
 
-The following section highlights deprecated features in {builds-shortname} 1.5.
-
 * With this update, the following channels are deprecated:
 ** builds-1.3
 
```

Comment:
I suggest you check this. Perhaps this misalignment can be fixed by adding the plus sign?
<img width="652" height="195" alt="image" src="https://github.com/user-attachments/assets/8ddd0f75-8da5-41b9-a250-237df35f9237" />

```suggestion
+
```